app/controllers/order_management/order_details_controller.rb

```diff
@@ -117,6 +122,10 @@ def load_order_statuses
     else
       @order_statuses = OrderStatus.non_protected_statuses(current_facility)
     end
+
+    if !@order_detail.unrecoverable? && cannot?(:mark_unrecoverable, OrderDetail)
```

Comment:
nit: I wonder if this `if` with `!` and `cannot?` could be clearer, but I don't really have a strong opinion

Reply:
I rewrote it with `unless`